posthog/hogql/database/models.py

```diff
@@ -95,6 +95,8 @@ def get_constant_type(self) -> "ConstantType":
 
 class ExpressionField(DatabaseField):
     expr: Expr
+    # Pushes the parent table type to the scope when resolving any child fields
+    isolate_scope: Optional[bool] = None
```

Comment:
Have had to add this because PoE relies on some Expression Fields that assume other tables are present outside of it's immediate scope. Probably bad practice to have PoE working like that, but it massively blows the scope of this to fix that